waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -916,7 +925,13 @@
            :resources {:limits {:memory memory}
                        :requests {:cpu cpu :memory memory}}
            :volumeMounts [{:mountPath "/srv/www"
-                           :name "user-home"}]})))))
+                           :name "user-home"}]}))
+
+      ; Optional reverse-proxy envoy sidecar
+      (pos-int? offset)
```

Comment:
This doesn't need to be conditional. We should always add this annotation.

Reply:
Done